tools/gulptasks/unsorted/update-devDependencies.js

```diff
@@ -0,0 +1,30 @@
+/* eslint no-console: 0 */
+const gulp = require('gulp'),
+    path = require('path');
+/**
+ * Update the updateDevDependencies files to resolve vulnerabilities
+ *
+ * @return {Promise<Array<*>>}
+ *         Promises to keep
+ */
+function updateDevDependencies() {
+    const { readFile, writeFile } = require('fs/promises');
+
+    const promises = [],
+        pathPartToNodeModules = [__dirname, '..', '..', '..', 'node_modules'];
+
+    // Update taffydb to @jsdoc/salty in outdated devDependencies
+    const publishPath = path.join(
+        ...pathPartToNodeModules, 'ink-docstrap', 'template', 'publish.js'
+    );
+    promises.push(readFile(publishPath).then(content => writeFile(
+        publishPath,
+        content.toString().replace(
+            'require(\'taffydb\')',
```

Comment:
Consider RegExp to catch 'em all.

Reply:
It just needs to import it correctly at the start - then the usage is identical ([docs](https://github.com/jsdoc/jsdoc/blob/main/packages/jsdoc-salty/README.md#use-salty-in-a-jsdoc-template)).